src/components/pages/NotFound.jsx

```diff
@@ -1,8 +1,12 @@
+import { Link } from "react-router-dom";
+import img from '../../assets/notFound.png';
+
 const NotFound = () => {
   return (
-    <>
-      <h1>404<br />Page Not Found</h1>
-    </>
+    <div className="bg-lightGreen w-full h-screen flex flex-col items-center pt-5">
+      <img src={img} alt="Error 404" height="350px"/>
```

Comment:
Let's use Tailwind CSS class for consistency  className="h-[350px]" instead of height="350px". You will notice this not takes effect with the height of your image. Readjust to your liking.

Reply:
Got it. Thank you